icons/database-backup.svg

```diff
@@ -10,9 +10,9 @@
   stroke-linejoin="round"
 >
   <ellipse cx="12" cy="5" rx="9" ry="3" />
-  <path d="M3 12c0 1.18 2.03 2.2 5 2.7" />
-  <path d="M21 5v4.5" />
-  <path d="m12 16 1.27-1.35a4.75 4.75 0 1 1 .41 5.74" />
-  <path d="M12 12v4h4" />
-  <path d="M3 5v14c0 1.43 2.97 2.63 7 2.93" />
+  <path d="M7 14.5C4.6 14 3 13 3 12" />
+  <path d="M9.5 21.9C5.8 21.6 3 20.4 3 19V5" />
+  <path d="M21 5v4" />
+  <path d="m11 16 2.5-2.5c1-1 2.3-1.5 3.5-1.5 2.8 0 5 2.2 5 5s-2.2 5-5 5c-1.6 0-3.1-.8-4-2" />
```

Comment:
Hm. Maybe this have a bit better curvature?
```suggestion
  <path d="M13 20c.9 1.2 2.4 2 4 2 2.8 0 5-2.2 5-5s-2.2-5-5-5c-2.37 0-4.45 1.59-6 4" />
```
![image](https://user-images.githubusercontent.com/17746067/231521216-cf7d9707-57a3-48d0-a8a6-ba04e7a1d8bc.png)
The difference is quite subtle on 100%, but it's there:
![image](https://user-images.githubusercontent.com/17746067/231521450-95486c37-1263-49dc-a956-2aec143bb604.png)

Reply:
Well this is how wonky it currently looks, so either way is better than this…

<img width="452" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/231525677-d415ac53-76ed-43e1-a25c-dcf2ac9b375f.png">

I personally prefer the more geometric version, which also seems to match the existing `refresh` and other icons… Either way, they should all be consistent.

Reply:
> Maybe this have a bit better curvature?

So other arrows in this style (see https://lucide.dev/icons?search=ccw) don’t have this proposed extra curvature… I was going for consistency with the existing version… I think either we have to change them all, or stick with the existing approach… Thoughts? @karsa-mistmere @ericfennis @jguddas

Reply:
@danielbayley @karsa-mistmere I designed these. That there is no curvature on the end is on purpose. 
I can give you an example with the current refresh icons, these also haven't curved arrow ends (which I designed as well). To be short on this answer: It looks better on 100% scale. Which is more important than that the icon will look good when zoomed in.

Here is a visual comparison that explains it.
<img width="659" alt="image" src="https://user-images.githubusercontent.com/11825403/235089287-75332a75-36ee-4346-9dc5-611bbb4825d4.png">

Reply:
@ericfennis Yeah I thought the same, which is why I matched this to the existing ones. Should be good to go then…? @karsa-mistmere

Reply:
I did provide a 100% view, the left one is the current, the right one is with the proposed geometric curve:
![image](https://user-images.githubusercontent.com/17746067/231521216-cf7d9707-57a3-48d0-a8a6-ba04e7a1d8bc.png)

To me, the geometric curve looks cleaner at 100%. 😅

Reply:
@ericfennis: This isn't perfect, but it's very very close to the current icon but has a less wonky path :
![database-backup](https://user-images.githubusercontent.com/17746067/235162691-5d6df350-f90d-44bb-8e29-f3d3534de039.svg)

![image](https://user-images.githubusercontent.com/17746067/235163394-d240298b-79a3-4d2f-b959-9b87b4b7079f.png)
![image](https://user-images.githubusercontent.com/17746067/235163118-cebc92b2-f2ed-4000-969a-183821d0fa47.png)

Reply:
@karsa-mistmere @ericfennis ccw arrow now perfectly matches #1053…

Reply:
@karsa-mistmere Dirty trick, but it kinda works… 😅

<img width="330" alt="Untitled" src="https://user-images.githubusercontent.com/7797479/235307585-00517981-94ec-4b1d-b4dd-5a7976bedeba.png">

Reply:
@danielbayley awesome!